### ISSUE TITLE
SWARM_DISCOVERY env variable doesn't work

### ISSUE THREAD

**Issue by phemmer** [docker/swarm#257]
```
# SWARM_DISCOVERY="file://$(pwd)/.swarm_discovery" swarm manage --host 127.0.0.1:2375
FATA[0000] --discovery required to manage a cluster     

# cat .swarm_discovery
172.17.0.13:2375
172.17.0.14:2375
172.17.0.15:2375
```

This was working at one point as I've used it before.

**Comment by vieux** [comment]
I see `SWARM_DISOVERY` in your gist, missing the `C`

**Comment by vieux** [comment]
But you're right, I'm looking into it

**Comment by phemmer** [comment]
Oops, typo while creating a simple reproduction instructions :-)

**Comment by vieux** [comment]
@phemmer fixed in 829bcb562d2df0ba7a0560bf480a6dcd30361df8 Thanks for the report.
